lib/physical/shipment.rb

```diff
@@ -7,17 +7,23 @@ class Shipment
                 :destination,
                 :service_code,
                 :pallets,
+                :structures,
                 :packages,
                 :options
 
-    def initialize(id: nil, origin: nil, destination: nil, service_code: nil, pallets: [], packages: [], options: {})
+    def initialize(id: nil, origin: nil, destination: nil, service_code: nil, pallets: [], structures: [], packages: [], options: {})
       @id = id || SecureRandom.uuid
       @origin = origin
       @destination = destination
       @service_code = service_code
-      @pallets = pallets
+      @structures = Set[*structures]
```

Comment:
Should this not be ordered? Like, when we iterate over it, do we want to guarantee the order?

Reply:
Maybe so. I was making it a set because that's how items on packages were being handled, but I don't have a solid reason for doing so outside performance. Do you think packages within structures should be ordered as well? What was the original reason for making package items a set? Just performance?

Reply:
We paired on this, and it's better as an Array.

Reply:
After discussing IRL we decided to make this an array so we get ordering.